libxrdp/libxrdpinc.h

```diff
@@ -98,13 +102,14 @@ struct display_size_description;
 /***
  * Initialise the XRDP library
  *
- * @param id Channel ID (xrdp_process* as integer type)
+ * @param proc XRDP instance to use with this library
```

Comment:
```suggestion
 * @param id XRDP instance to use with this library
```

The parameter is still called `id`.